Gemfile

```diff
@@ -61,7 +61,7 @@ end
 if ENV['RAILS_VERSION']&.match(/7\.0\.0/) || !ENV['RAILS_VERSION']
   gem 'paranoia', github: 'payrollhero/paranoia', branch: 'rails7'
   gem 'awesome_nested_set', github: 'damianlegawiec/awesome_nested_set', branch: 'master'
-  gem 'ransack', github: 'activerecord-hackery/ransack', branch: 'master'
+  gem 'ransack', github: 'activerecord-hackery/ransack', branch: 'main'
```

Comment:
totally forgot - we need to use ransack 2.x, also we don't need to use the github version anymore, release with rails 7 support was released to rubygems some time ago